action.yml

```diff
@@ -38,6 +44,16 @@ runs:
           if [[ $flag = true ]]; then
             if [[ $is_pr = true ]]; then
               gh pr comment $issue --body "$message"
+            elif [[ $is_discussion = true ]]; then
+              gh api graphql -F discussionId="$discussion_node_id" -F body="$message" -f query='
+                mutation($discussionId: ID!, $body: String!) {
+                  addDiscussionComment(input: {discussionId: $discussionId, body: $body}) {
```

Comment:
`replyToId` (The Node ID of the discussion comment within this discussion to reply to.) might be useful when we add `discussion comments`